modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -92,13 +93,23 @@ function dump_mongo {
 
 function restore_mongo {
   cd "${tempdir}"
-  tar --extract --gzip --file "${filename}"
+  tar --extract --gzip --force-local --file "${filename}"
```

Comment:
The [reason behind `--force-local`](https://steve-parker.org/linux/tar-colon-filename/) sounds like _fun_.

Reply:
I'd prefer if the `--force-local` argument addition in each of these cases was split out into a separate commit, but that might just be me being picky.

Reply:
it is done :)